DirectGuiExtension/DirectCollapsibleFrame.py

```diff
@@ -43,20 +39,34 @@ def __init__(self, parent = None, **kw):
             text=self['extendText'] if self['collapsed'] else self['collapseText'],
             text_scale=0.05,
             text_align=TextNode.ALeft,
-            text_pos=(DGH.getRealLeft(self)+0.02, DGH.getRealTop(self)-self['headerheight']/2.0),
+            text_pos=(frameSize[0]+0.02, frameSize[3]-self['headerheight']/2.0),
             borderWidth=(0.02, 0.02),
             relief=DGG.FLAT,
             pressEffect=False,
             frameSize = (
-                DGH.getRealLeft(self), DGH.getRealRight(self),
-                DGH.getRealTop(self)-self['headerheight'], DGH.getRealTop(self)),
+                frameSize[0], frameSize[1],
+                frameSize[3]-self['headerheight'], frameSize[3]),
             command=self.toggleCollapsed)
 
+        # Call option initialization functions
+        self.initialiseoptions(DirectCollapsibleFrame)
+
+        self.resetFrameSize()
+        self.updateFrameSize()
+
+        self.originalFrameSize = self['frameSize']
+
+    def addItem(self, element, *args):
```

Comment:
This seems unfinished since it's not adding anything to this widget but only print the elements and args. What did you want to use this for?

Reply:
Whops, this was only meant for testing. I will remove that.